tests/helm/testdata/goldenfile/logs_otc/basic.output.yaml

```diff
@@ -213,6 +212,7 @@ data:
         - update-ca-certificates.service
         - user-cloudinit.service
         - var-lib-etcd2.service
+        - kubelet.service
```

Comment:
just curious, is there a reason to move ```kubelet.service``` to the end? looks like this was in alphabetical order 🤔

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3845/files#diff-bd3221290637efd3855e2de7717869ccb4e6eb4cf2dcc177729dd61e9cb2fe06R121

In config yaml since we have splitted systemd and kubelet with separate flags, all systemd units come first and kubelet after those. Previously since kubelet was as subset of systemd, we have kept systemd units in alphabetical order. 
@chan-tim-sumo